src/utils/DepositUtils.ts

```diff
@@ -71,3 +73,38 @@ export async function queryHistoricalDepositForFill(
 
   return validateFillForDeposit(fill, deposit) ? deposit : undefined;
 }
+
+export function createDepositForSimulatingGas(
+  amountToRelay: BigNumberish,
+  tokenSymbol: string,
+  originChainId: string,
+  destinationChainId: string,
+  relayerAddress: string,
+  messagePayload?: {
+    message: string;
+    recipientAddress: string;
+  }
+): Deposit {
+  const originToken = resolveContractFromSymbol(tokenSymbol, originChainId);
+  const destinationToken = resolveContractFromSymbol(tokenSymbol, destinationChainId);
+  if (!isDefined(originToken) || !isDefined(destinationToken)) {
+    throw new Error(`Could not resolve token contract for ${tokenSymbol} on ${originChainId} or ${destinationChainId}`);
+  }
+
+  return {
+    // We want to set the deposit ID to 0 so that there
+    // is no chance of a collision with a real deposit
+    depositId: 0,
```

Comment:
What's wrong with a collision?

Reply:
This was a result of a conversation I had with @pxrl. He mentioned to avoid potentially causing collisions in the test

Reply:
I don't understand why we need to avoid collisions can you explain more?

Reply:
The issue with an exact collision is that the relay could already be filled, resulting in a revert. A collision is unlikely, though, as all the other params would have to line up. Agreed on avoiding them, though, by making the deposit id unreachable.

Were there deposits with deposit id 0? I forget. If not, 0 is fine.

Note: if people know this number, they could technically try to trigger a collision by sending bad relays. A fix for that would be to hide it via env, but I think this is unlikely and hard to pull off given all the other params would need to match, too.

Reply:
Sounds good - I believe we should be fine with 0.

Reply:
hmm i guess `MaxUint32` is safer but might cause some errors in the clients. Have you tested that they can handle that large of a deposit id?

Reply:
The `depositId` in the `SpokePool.sol` is defined as a `uint32` so I figure we could max it out. Technically we could probably get away with setting a `depositId = -1` which is going to underflow to somewhere around the max int.

Reply:
i think that's a tad bit safer than `0`

Reply:
Done: https://github.com/across-protocol/sdk-v2/pull/427/commits/831cf21cb93d8f43981345517bd81815bbf4076a